manifests/install.pp

```diff
@@ -26,8 +26,14 @@
     } else {
       $provider = undef
     }
+
+    $manage_package = $::jmxtrans::package_version ? {
+      ''      => present,
+      default => $jmxtrans::package_version,
+    }
+
     package { $::jmxtrans::package_name:
-      ensure   => present,
+      ensure   => $manage_package,
```

Comment:
After setting the default value for `$::jmxtrans::package_version` to `'present'`, this can just be `ensure => $::jmxtrans::package_version`.